Several tests expect specific terminal size.

Not intended for upstream.

--- oslo_config-10.2.0/oslo_config/tests/test_cfg.py
+++ oslo_config-10.2.0/oslo_config/tests/test_cfg.py
@@ -142,6 +142,7 @@ class BaseTestCase(base.BaseTestCase):
 
 class UsageTestCase(BaseTestCase):
 
+    @unittest.skip("expects specific terminal size")
     def test_print_usage(self):
         f = io.StringIO()
         self.conf([])
@@ -166,6 +167,7 @@ class UsageTestCase(BaseTestCase):
         self.assertNotIn('tepilog', f.getvalue())
         self.assertNotIn('optional:', f.getvalue())
 
+    @unittest.skip("expects specific terminal size")
     def test_print_help(self):
         f = io.StringIO()
         self.conf([])
@@ -181,6 +183,7 @@ class UsageTestCase(BaseTestCase):
 
 class HelpTestCase(BaseTestCase):
 
+    @unittest.skip("expects specific terminal size")
     def test_print_help(self):
         f = io.StringIO()
         self.conf([])
